pkg/gohai/utils/helpers.go

```diff
@@ -0,0 +1,15 @@
+// This file is licensed under the MIT License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2014-present Datadog, Inc.
+
+package utils
+
+import "bytes"
+
+// StringFromBytes converts a null-terminated (C-style) string to a Go string
+//
+// The given slice must be null-terminated.
+func StringFromBytes(slice []byte) string {
+	length := bytes.IndexByte(slice, 0)
+	return string(slice[:length])
+}
```

Comment:
If the slice is null-terminated why not use `string(slice)` ?

Reply:
Actually if you just use `string(slice)` the string will contain the 0 bytes, you might not see them when they're printed but they're still here. You can check that by running 
```
func main() {
	s := [5]byte{65, 66, 67, 0, 0}
	fmt.Println(string(s[:]))
}
```
and redirecting the output to a file, then running `xdd -p <file>` (or using hexdump) you'll see the 0 bytes (ie. `41424300000a` which is `A B C null null newline`).

I looked online and the only way I could find to avoid that was to explicitly cut at the first null byte.

Reply:
OK, good to know, mind adding this as a comment for `StringFromBytes` ? So other don't wonder why we're not using `string()` too.